.gitignore

```diff
@@ -1,2 +1,19 @@
-node_modules/
-.DS_Store
\ No newline at end of file
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
```

Comment:
Let's not get into the habit of committing code which is only relevant for our own workflow. Add all of the #misc stuff to your own global .gitignore, and remove them here. Make sense?